app/views/users/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>This is the home page</h1>
+<ul>
+	<% @users.each do |user| %>
+		<li>
+			<%= link_to user.email, user %>
```

Comment:
is this working?
Please also add screenshots of all the pages.

Reply:
yes its working. will add screenshots

Reply:
![image](https://user-images.githubusercontent.com/110826076/188426544-a75187c1-66a8-45c3-b4d7-e0c4f7483444.png)